addons/website_slides/tests/test_slide_channel.py

```diff
@@ -274,6 +275,43 @@ def _get_completion_time_pdf(*args, **kwargs):
         self.assertEqual(13.37, round(slides_1.completion_time, 2))
         self.assertEqual(123.0, slides_2.completion_time)
 
+    def test_if_unlink_and_unpublish_channel_slides_sends_competition_email(self):
```

Comment:
Consistency with other related tests
```suggestion
    def test_mail_completed_not_on_unpublishing_or_unlinking_slides(self):
```